tests/Test/AbstractFixerTestCase.php

```diff
@@ -703,6 +706,34 @@ private static function assertValidDescription(string $fixerName, string $descri
         self::assertFalse(strpos($descriptionType, '``'), \sprintf('[%s] The %s must no contain sequential backticks.', $fixerName, $descriptionType));
     }
 
+    private static function assertOptionDefault(FixerOptionInterface $option, FixerInterface $fixer): void
+    {
+        if (!$option->hasDefault()) {
+            return;
+        }
+
+        $allowedValues = $option->getAllowedValues();
+
+        if (!is_countable($allowedValues) || 1 !== \count($allowedValues)) {
```

Comment:
```suggestion
        if (null !== $allowedValues) {
```

by definition, it's either array or null. no need for `is_countable`